### PR TITLE
bugfix search autocomplete

### PR DESCRIPTION
**Pull request by nemanjadimsic**
if everything is fine, issue https://github.com/mkitbs/ZAGA-WEB/issues/4 is to be closed

### CODE REVIEW

File: zaga-web/src/app/components/creatework-order/creatework-order.component.ts [hunk 1]
```diff
@@ -85,233 +86,255 @@ export class CreateworkOrderComponent implements OnInit {
...
   selectedMaterial;
 
-  nameFC : FormControl = new FormControl("");
-  
+  nameFC: FormControl = new FormControl("");
+
   filteredOptions: Observable<string[]>;
 
   ngOnInit() {
     this.query = this.nameFC.value;
```
Comment (katic-milan) [inline zaga-web/src/app/components/creatework-order/creatework-order.component.ts:94]: What is this used for?

**Review by katic-milan — CHANGES_REQUESTED** [review]
I left comment in createWorkOrder.ts file.

File: zaga-web/src/app/components/creatework-order/creatework-order.component.ts (hunk 1 above)
Comment (nemanjadimsic) [reply zaga-web/src/app/components/creatework-order/creatework-order.component.ts:94]: Deprecated, will delete in the next commit

**Review by katic-milan — APPROVED** [review]
LGTM

**Pull request merged**